changelogs/fragments/5804-minor-changes-to-hpe-ilo-collection.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - ilo_redfish_config - Change input type of attribute_value (https://github.com/ansible-collections/community.general/pull/5804).
+  - ilo_redfish_utils - Change implementation of DNS Server IP and NTP Server IP update (https://github.com/ansible-collections/community.general/pull/5804).
```

The first fragment isn't needed, since the type doesn't change. (No type means `type='str'`.)
```suggestion
  - ilo_redfish_utils module utils - change implementation of DNS Server IP and NTP Server IP update (https://github.com/ansible-collections/community.general/pull/5804).
```